sdk/netapp/Azure.ResourceManager.NetApp/src/Custom/ArmNetAppModelFactory.cs

```diff
@@ -36,7 +36,7 @@ public static partial class ArmNetAppModelFactory
 
         public static NetAppBackupData NetAppBackupData(ResourceIdentifier id = null, string name = null, ResourceType resourceType = default, Azure.ResourceManager.Models.SystemData systemData = null, AzureLocation location = default, string backupId = null, DateTimeOffset? createdOn = null, string provisioningState = null, long? size = null, string label = null, NetAppBackupType? backupType = null, string failureReason = null, string volumeName = null, bool? useExistingSnapshot = null)
         {
-            return new NetAppBackupData(id, name, resourceType, systemData, backupId, createdOn, provisioningState, size, label, backupType, failureReason, null, useExistingSnapshot, snapshotName: null, backupPolicyArmResourceId: null,  serializedAdditionalRawData: null);
+            return new NetAppBackupData(id, name, resourceType, systemData, backupId, createdOn: createdOn, snapshotCreationOn: null, completionOn: null, provisioningState: provisioningState, size: size, label: label, backupType: backupType, failureReason: failureReason, volumeResourceId: null, useExistingSnapshot: useExistingSnapshot, snapshotName: null, backupPolicyArmResourceId: null, isLargeVolume:null, serializedAdditionalRawData: null);
```

Comment:
well - I think we would better change the implementation here to call the public method in the same class, so that in the future we do not ever need to modify this to make it compile.
```suggestion
            return NetAppBackupData(id, name, resourceType, systemData, backupId, createdOn: createdOn, provisioningState: provisioningState, size: size, label: label, backupType: backupType, failureReason: failureReason, useExistingSnapshot: useExistingSnapshot);
```
because all the parameters in the generated method in this model factory would be optional, we just need to include the parameters we currently have via positional arguments, and it should always be able to compile in any future versions.